### PR TITLE
editor layout changes

### PR DESCRIPTION
**Pull request by ezequielc**
@alex-deaconu Please review. Thanks

fyi @ghislaineguerin

### CODE REVIEW

File: web/partials/editor/presentation-properties-modal.html [hunk 1]
```diff
@@ -41,6 +41,12 @@ <h3 class="modal-title" translate>editor-app.properties.title</h3>
     </div>
 
   </form>
+
+  <!-- Indicates delete or destructive action -->
+  <button id="deleteButton" ng-show="presentationProperties.id" require-role="cp" type="button" class="btn btn-danger btn-fixed-width" ng-click="confirmDelete()">
+    {{'common.delete' | translate}} Presentation<i class="fa fa-white fa-trash-o icon-right"></i>
+  </button>
+
```
Comment (alex-deaconu) [inline web/partials/editor/presentation-properties-modal.html]: @ezequielc is the `Delete Presentation` button intended to be on a different row than the others: http://screencast.com/t/AgdrQ9jN9Jw? Thanks.

File: web/partials/editor/toolbar.html [hunk 2]
```diff
@@ -1,22 +1,26 @@
...
-    <div class="col-sm-12 col-md-7">
-      <div ng-init="designMode = true" class="btn-group btn-inverse pull-right">
-        <button id="designButton" ui-sref="apps.editor.workspace.artboard" class="btn btn-inverse" ng-class="{'active' : designMode}" ng-disabled="designMode" ng-click="designMode = true"  >{{'editor-app.workspace.toolbar.mode.design' | translate}}</button>
-        <button id="htmlButton" ui-sref="apps.editor.workspace.htmleditor" class="btn btn-inverse" ng-class="{'active' : !designMode}" ng-disabled="!designMode" ng-click="designMode = false" >{{'editor-app.workspace.toolbar.mode.html' | translate}}</button>
-      </div><!--btn-group-->
-      <button id="compressArtboardButton" class="btn btn-inverse animated pulse ng-hide" ng-click="fullArtboard = false" ng-show="designMode && fullArtboard && !placeholderFactory.placeholder"><i class="fa fa-compress"></i></button>
-      <button id="expandArtboardButton" class="btn btn-inverse" ng-show="designMode && (!fullArtboard || placeholderFactory.placeholder)" ng-click="fullArtboard = true; placeholderFactory.clearPlaceholder()"><i class="fa fa-expand"></i></button>
+  <div class="">
+    <div class="">
```
Comment (alex-deaconu) [inline web/partials/editor/toolbar.html]: @ezequielc are these empty `div`s necessary? Thanks.

File: web/partials/editor/presentation-properties-modal.html (hunk 1 above)
Comment (ezequielc) [inline web/partials/editor/presentation-properties-modal.html]: @ghislaineguerin Could you confirm @alex-deaconu question? Thanks!

File: web/partials/editor/toolbar.html (hunk 2 above)
Comment (ezequielc) [inline web/partials/editor/toolbar.html]: @alex-deaconu Yes, they are not necessary. I am removing them. Thanks!

Comment (alex-deaconu) [inline web/partials/editor/toolbar.html]: @ezequielc thanks!

**Comment by alex-deaconu** [comment]
@ezequielc reviewed, PR looks good :+1: . However, still waiting for an answer re Delete Presentation button location from @ghislaineguerin. Thanks.

**Comment by ezequielc** [comment]
Thanks @alex-deaconu .  I will wait. Cheers!

**Comment by ghislaineguerin** [comment]
@alex-deaconu yes, it is meant to be on a different row.

**Comment by ezequielc** [comment]
Thanks @ghislaineguerin ! Can I merge @alex-deaconu ? Cheers

**Comment by alex-deaconu** [comment]
@ezequielc yup. @ghislaineguerin thanks for the confirmation!

**Pull request merged**